workflow/table_generation.py

```diff
@@ -0,0 +1,800 @@
+# -*- coding: utf-8 -*-
+"""
+Utility functions for processing Wolf-Yadlin EGFR proteomic data.
+Created on 1/12/15
+
+editited (for python 3) by Adam Shedivy 05/22/2020
+
+@author: Anthony Gitter
+"""
+
+from collections import defaultdict
+from functools import partial
+import itertools
+import numpy as np
+import pandas as pd
+
+# A copy of PrepTemporalCytoscape that is modified to work with TPS input
+# and output files
+# TODO: make the log transformation more robust, handle 0s without requiring
+# a default value
+
+
+def PrepTemporalCytoscapeTPS(peptideMapFile, timeSeriesFile, peptideFirstScoreFile,
```

Comment:
There are many places where this code could be improved or modernized, but we should focus on getting a functional version first.  Then we can add test cases before making changes to this implementation.